packages/grid/grid.0.1.0/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "2.0"
+maintainer: "Jean-Christophe.Filliatre@cnrs.fr"
+authors: "Jean-Christophe Filliâtre"
+synopsis: "A tiny library for two-dimensional arrays"
+description: "Elementary functions over two-dimensional arrays. Not a linear algebra library."
+license: "LGPL-2.1"
+homepage: "https://github.com/backtracking/grid"
+doc: "https://backtracking.github.io/grid"
+bug-reports: "https://github.com/backtracking/grid/issues"
+depends: [
+  "ocaml"
+  "dune" {>= "2.0.0"}
+]
+build: [
+  ["dune" "subst"] {pinned}
```

Comment:
```suggestion
  ["dune" "subst"] {dev}
```

This is likely due to [a bug that is fixed in Dune 2.7](https://github.com/ocaml/dune/blob/main/CHANGES.md#270-13082020)

Reply:
fixed